src/config/constants.js

```diff
@@ -0,0 +1,4 @@
+const constants = {
+  brandName: 'UndocuScholars',
+};
+export default constants;
```

Comment:
I recommend disabling the single export per file rule here and just exporting the constant:
```suggestion
/* eslint-disable import/prefer-default-export */
export const BRAND_NAME = 'Undocuscholars';
```
Otherwise it's possible to set `constants.brandName` even though it should be a constant (the only thing constant about a JS object is the reference).

Also, per [23.10](https://airbnb.io/javascript/#naming--uppercase) of the AirBnB style guide exported constants should be uppercased. It helps visually see when a constant is being used.

Reply:
I changed it because of the lint error but now that I know how to disable it, I'll revert it

Reply:
Yeah, the rule is there to help encourage single exports per file but it doesn't make sense in the case of this file.